src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx

```diff
@@ -3,51 +3,8 @@ import {shallow} from '@jahia/test-framework';
 import ContentHeader from './ContentHeader';
 
 describe('Content header', () => {
-    it('Should render with everything', async () => {
```

Comment:
Nope, don't delete tests

Reply:
I update the component, so I must delete and or update the test :D

Reply:
Yes please update the test to the new behaviour.

All test that you delete, you can move them to another place where the unchanged test belong.

Reply:
the is no other places, the code doesn't exist anymore